### DISCUSSION TITLE
MouseWarpOverride & WineCfg grab mouse doesn't work in GTA IV

### DISCUSSION THREAD

**Discussion by Gravarty** [The-Wineskin-Project/WineskinServer#252] [General]
In GTA IV, the mouse does not grab ingame and continues to display the standard macOS cursor. You can still play the game, but when the cursor reaches the end of the screen, you have to move the mouse really fast to turn the camera, and that makes it really hard to play.

It doesn't happen with other games. Maybe this is just a issue with Wine3D games?

**Comment by Gcenx** [comment]
Open the game full screen and give it accessibility permissions so it can grab the mouse cursor.

You may need to manually give this permissions as you’ll nightly have missed the prompt due to being fullscreen.

**Reply by Gravarty** [reply]
Oh, sorry. I completely forgot to say that the accessibility permissions are already set to allow.
The game only supports full screen and not borderless.

**Reply by Gcenx** [reply]
I’m not sure what’s going on then as that’s usually the fix, maybe the new engine will help as CX21.2 back-ported the use of apples private mouse capture api.

**Comment by Gravarty** [comment]
Sorry, i know it's been a while but this issue still exists. GTA IV is so far the only game that just won't grab the mouse in game or in menus. 

All of the accessibility permissions are set and override is also set to active.
